generator/AWSPSGeneratorLib/HelpMaterials/Examples/S3/Get-S3CORSConfiguration.xml

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$CORSConfig = Get-S3CORSConfiguration -BucketName 's3testbucket'
+$CORSConfig.Rules
```

Comment:
`PS C:\>` usage for commands v/s output - where both of them are showing together - it can be confusing.